### PR TITLE
Added Hillstone StoneOs detection

### PR DESCRIPTION
**Pull request by alangregory**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/definitions/stoneos.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        text: Device Traffic
+    -
+        graph: device_processor
+        text: CPU Usage
+    -
+        graph: device_mempool
+        text: Memory Usage
+discovery:
+    -
+        sysObjectId: .1.3.6.1.4.1.28557.1
```
Comment (laf) [inline includes/definitions/stoneos.yaml]: Do you need sysDescr as well? Will sysObjectID not be unique enough?

File: includes/definitions/stoneos.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+os: stoneos
+text: Hillstone StoneOs
+type: network
+icon: hillstone
+group: hillstone
+over:
```
Comment (laf) [inline includes/definitions/stoneos.yaml]: Can you format the over graphs like the other yaml files please.

File: includes/definitions/stoneos.yaml (hunk 1 above)
Comment (alangregory) [reply includes/definitions/stoneos.yaml]: I think only Hillstone use this Oid, removed sysdescr match.

File: includes/definitions/stoneos.yaml (hunk 2 above)
Comment (alangregory) [reply includes/definitions/stoneos.yaml]: Fixed, was my yaml format plugin that changed it.

**Comment by murrant** [comment]
@alangregory You need to reduce your svg files.

You can use this site: https://jakearchibald.github.io/svgomg/

**Comment by laf** [comment]
@alangregory Thanks for submitting this :)

Would you mind submitting some additional test data as per https://docs.librenms.org/#Developing/os/Test-Units/#example-workflow

Ensure you anonymise the data before you submit it though.

**Pull request merged**